thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -81,6 +81,7 @@
             anomalyRange=context.anomalyRange
             displayRange=context.analysisRange
             compareMode=context.compareMode
+            timeseries=timeseries
```

Comment:
does this belong in this PR?

Reply:
Its for the next PR :)

Reply:
Great minds think alike :) I'll put this away for now.